mmv1/third_party/terraform/tests/resource_compute_instance_template_test.go.erb

```diff
@@ -1205,16 +1205,20 @@ func TestAccComputeInstanceTemplate_sourceSnapshotEncryptionKey(t *testing.T) {
 
 	var instanceTemplate compute.InstanceTemplate
 	kmsKey := BootstrapKMSKeyInLocation(t, "us-central1")
-	kmsKeyName := GetResourceNameFromSelfLink(kmsKey.CryptoKey.Name)
-	kmsRingName := GetResourceNameFromSelfLink(kmsKey.KeyRing.Name)
+
+	context := map[string]interface{}{
+		"kmsRingName":        GetResourceNameFromSelfLink(kmsKey.CryptoKey.Name),
```

Comment:
Nit: spacing between the key and value is a little weird here and in the other context block

Also, I would stick with a consistent capitalization for the keys here. For example, either `kms_ring_name`, or `randomSuffix`

Reply:
Totally Agree sorry I missed this